pkgs/applications/audio/netease-cloud-music-gtk/default.nix

```diff
@@ -0,0 +1,79 @@
+{ lib
+, stdenv
+, glib
+, gtk3
+, curl
+, dbus
+, openssl
+, gst_all_1
+, pkg-config
+, rustPlatform
+, wrapGAppsHook
+, fetchurl
+, fetchFromGitHub
+, makeDesktopItem
+}:
+let
+  pname = "netease-cloud-music-gtk";
```

Comment:
since you are doing `rec {` below you can just move this directly into the derivation